file-formats/desd/examples/z_aff_example_desd.desd.json

```diff
@@ -0,0 +1,10 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example DESD for ABAP file format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
+  },
+  "generalInformation": {
```

Comment:
I saw that the example object Z_AFF_EXAMPLE_DESD does not exist on the database and you only took the json from the Test Tool. Can you please create the object and use programm SAFF_GENERATE_REPO to regenerate the repo folder.

After you regenerated the repo folder, the field abapLanguageVersion will no longer be shown when the value is standard, because that is the default. Also the object generalInformation will no longer be in the json when it and all of its subfields are initial.

Reply:
@JoergCMaier mentioning here for visibility

Reply:
Hi @huber-nicolas, should be solved. BR, Joerg

Reply:
@JoergCMaier Hi, this example json needs to be updated, as Nicolas has mentioned in the second part of the comment